test/EndToEnd/Ledger.hs

```diff
@@ -62,18 +64,34 @@ testSuite :: TestTree
 testSuite =
   testGroup
     "L1"
-    [ testCase "bidder-buys" bidderBuysTest
+    [ testCase "bidder-buys" bidderBuysNoDepositTest
     , testCase "seller-reclaims" sellerReclaimsTest
     , testCase "seller-bids" sellerBidsTest
     , testCase "unauthorised-bidder" unauthorisedBidderTest
+    , testGroup "bidder-deposit" bidderDepositTests
     ]
 
+bidderDepositTests :: [TestTree]
```

Comment:
Please separate them to another module. This one gets too big now, and you already separated new group.